test_util.go

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
we can just have all the code in one file instead of two

Reply:
I highly recommend against that! Notice that util_test.go (was: test_util.go) imports "testing". We do NOT want to import testing in the production DB!  In fact, this is why I renamed from test_util.go to util_test.go: so that the file would be skipped during normal compilation and only included when doing testing.